src/ray/protobuf/experimental/autoscaler.proto

```diff
@@ -160,6 +156,16 @@ message GetClusterResourceStateReply {
   repeated ClusterResourceConstraint cluster_resource_constraints = 6;
 }
 
+message PendingInstanceRequest {
+  // The instance type name for the cloud, e.g. t1.micro
+  string instance_type_name = 1;
+  // The node type name, e.g. ray-head-node, matching `available_node_types`
+  // in the autoscaler config. See `ray/autoscaler/ray-schema.json`
+  string ray_node_type_name = 2;
+  // The number of such instance requested to be started.
+  int32 count = 3;
+}
+
 message ReportAutoscalingStateRequest {
```

Comment:
Can we wrap the fields under ReportAutoscalingStateRequest into a separate proto message, something like `AutoscalingState`? I feel it's a bit awkward to directly use these autoscaler rpc request/response on the product side.

See https://github.com/anyscale/product/pull/21538/files, thoughts @brucez-anyscale @scv119

Reply:
Agree. It is a good idea.